pyro/contrib/easyguide/easyguide.py

```diff
@@ -18,8 +17,7 @@
 from pyro.poutine.util import prune_subsample_sites
 
 
-@add_metaclass(ABCMeta)
-class EasyGuide(object):
+class EasyGuide(object, metaclass=ABCMeta):
```

Comment:
We can also use [ABC](https://docs.python.org/3/library/abc.html#abc.ABC) as the base class directly with python 3.4+.

Reply:
Will address in follow up, I copied this pattern from the six documentation

Reply:
this syntax seems to be for python 3.6+ but i was thinking that we would support python 3.4+ WDYT?